src/components/media/UserMedia.tsx

```diff
@@ -45,6 +46,15 @@ export default function UserMedia ({ index, uid, imageInfo, onClick, tagList, on
   const loader = isDesktop ? DesktopPreviewLoader : MobileLoader
   const shareableUrl = `/p/${uid}/${basename(imageInfo.filename)}`
 
+  const getUploadDate = (dateUploadedStr: string): string => {
```

Comment:
The build is failing in Vercel ([see log](https://vercel.com/openbeta-dev/open-tacos/EDmc8cYWDeLHqKyBVALsisvmd3UZ)).  I believe `ctime` is already a Date object.  
> Type error: Argument of type 'Date' is not assignable to parameter of type 'string'.
07:45:02.493

FYI now that build env issues have been resolved if you do a normal `git commit` without --no-verify, the pre-commit hook will run a full build locally and can catch these types of issues.  
Thank you for your patience!

Reply:
Oops, I missed that. Thanks for fixing the build env issues. I made an edit, ran the pre-commit hook, and the build was successful for me locally. However, I could not view this in the browser since I needed to remove the SIRV image hosting keys from my local.env.